source/content/nextjs/limitations.md

```diff
@@ -0,0 +1,124 @@
+
+We will create the same capacity for Next.js to clear CDN caches itself in response to appropriate events like [revalidateTag](https://nextjs.org/docs/app/api-reference/functions/revalidateTag). If you have thoughts or feedback please share it here in [this thread specific to the question of tag-based clearing](https://github.com/pantheon-systems/documentation/issues/9762) or [the general question of deployment adapters](https://github.com/pantheon-systems/documentation/issues/9732).
+
+### HTTP streaming
+
+Layers of our CDN and load balancing currently prevent HTTP Streaming for WordPress, Drupal, and Next.js. We introduced that limitation many years ago because we wanted to encourage teams to use full page caching in combination with Surrogate Keys for fine-grained purging. In WordPress and Drupal, that approach to CDN caching is accommodated by our [Pantheon Advance Page cache plugin](https://wordpress.org/plugins/pantheon-advanced-page-cache/) and [module](https://www.drupal.org/project/pantheon_advanced_page_cache).
+
+For many teams this restriction is counterproductive. That is especially true in the Next.js ecosystem which is investing further in usage of  [\<Suspense\>](https://react.dev/reference/react/Suspense) components as a performance optimization.
+
+While we intend to remove the limitation on streaming for Next.js sites, [join the discussion in this GitHub issue](https://github.com/pantheon-systems/documentation/issues/9767) if you have thoughts on how to provide guidance around situations where full page caching in the CDN. is still preferable to streaming.
+
+## General
+
+### Compatibility and Requirements
+See the following page for Next.js compatibility and requirements on Pantheon:
+* [Next.js Overview](/nextjs#compatibility--requirements)
+
+### **Bun, Deno, and other runtimes beyond Node.js**
+
+Node.js is the most common run time for Next.js. Bun and Deno both have compelling performance and security advantages that may make them preferable for some teams. If you want Pantheon to offer Bun, Deno, or any other runtime for JavaScript/TypeScript, please let us know [when you fill out the Beta request form](https://pantheon.io/nextjs-beta).
+
+### **Astro, Remix, and other frameworks beyond Next.js**
+
+Pantheon began many years ago as a Drupal-only platform. But the nature of our free trial allowed many customers to try other LAMP stack frameworks, many of which worked. In 2014 we made our support for WordPress official with very minimal modification to our tech stack. We made that policy change because our ecosystem has so much overlapping usage between WordPress and Drupal and because we value the success of web teams, and the web as a whole over the success of any given framework.
```

Comment:
Let's end this sentence at "official". There are more than enough layers at which details change for WordPress and Drupal that we shouldn't downplay the distinction.